### PR TITLE
Convert CI from Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by krievley**
Added the yml file to run the travis process on github actions.

### CODE REVIEW

File: .github/workflows/run-php-tests.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Run PHP Tests
+
+on:
+  push:
+    branches: [ krc/add-github-actions ]
```
Comment (mattstauffer) [inline .github/workflows/run-php-tests.yml:5]: You'll want to update this to `main` before we merge

File: .github/workflows/run-php-tests.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      with:
+          php-version: ${{ matrix.php }}
+          extensions: posix, dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, bcmath, soap, intl, gd, exif, iconv, imagick
+          coverage: none
+    
+    - name: Composer Self Update
+      run: composer self-update
+    
+    - name: Composer Update
+      run: composer update ${{ matrix.composer_flag_string }} --no-interaction --prefer-source
```
Comment (mattstauffer) [inline .github/workflows/run-php-tests.yml:41]: It looks like this isn't actually running the tests?

**Review by mattstauffer — CHANGES_REQUESTED** [review]

**Comment by mattstauffer** [comment]
Please delete any Travis files while you're at it, and add the GH Actions badge to the readme

**Comment by krievley** [comment]
Matt, per your feedback I have tried adding better commit messages and performed the updates you requested:

1. Changed the branch to watch to "main".
2. Added a PHPUnit step to the config.
3. Removed the Travis CI file.
4. Added GitHub Actions Badge to the README

**Comment by mattstauffer** [comment]
Thanks! One remaining note on commits: Please use the present imperative tense, so "Add" instead of "Added", "Create" instead of "Created", etc.

Additionally, you left the Travis badge still in the readme.

Please take the time to read each PR you make before submitting it so you can see if there's anything you could potentially catch before it makes it to the reviewer. Thanks!

**Comment by mattstauffer** [comment]
Thanks @krievley!!

**Pull request merged**